frontend/src/components/Course.tsx

```diff
@@ -24,11 +23,10 @@ function Course(props: any) {
 
   const handleStar = (courseId: any) => {
     if (isAuthed) {
-      // eslint-disable-next-line no-underscore-dangle
       starCourseAction(courseId);
     } else {
       // Force the user to go login // FIX I hate this feature. Let's make a modal instead
```

Comment:
Feel free to cut off the end of the comment now that this is changed

Reply:
good catch